ksql-common/src/main/java/io/confluent/ksql/util/KsqlTopicAuthorizationException.java

```diff
@@ -19,16 +19,17 @@
 
 import org.apache.commons.lang3.StringUtils;
 import org.apache.kafka.common.acl.AclOperation;
+import org.apache.kafka.common.errors.TopicAuthorizationException;
 
-public class KsqlAuthorizationException extends RuntimeException {
+public class KsqlTopicAuthorizationException extends TopicAuthorizationException {
```

Comment:
Also, could you move the class to the `ksql-engine` module? I see all the kafka exceptions are in the `ksql-engine/src/main/java/io/confluent/ksql/exception`